tests/zfs-tests/include/libtest.shlib

```diff
@@ -2843,3 +2843,30 @@ function block_device_wait
 		$UDEVADM settle
 	fi
 }
+
+function kill_process
+{
+	typeset -l i killpid=$1
+
+	#attempt to kill
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_must $KILL $killpid
+	fi
+	#verify killed worked
+	if $PS --no-headers -p $killpid > /dev/null; then
+		log_mustnot $KILL $killpid
```

Comment:
I have this in the routine will push into PR once I have added file_write as well